### DISCUSSION TITLE
David Rosenthal article/talk summary

### DISCUSSION THREAD

**Discussion by marypcbuk** [rufuspollock/awesome-crypto-critique#85] [General]
I love the collection of resources you have here; could I suggest an addition? https://blog.dshr.org/2022/02/ee380-talk.html Having built a decentralized consensus system using Proof-of-Work (http://dx.doi.org/10.1145/945445.945451) the author has the technical knowledge to explain the design faults and limitations of permissionless blockchain systems, as well as highlighting the economic and environmental issues. This might be the single best critique of web3 and cryptocurrencies I've read.

**Comment by spookylukey** [comment]
There are lots more great articles on his blog - https://blog.dshr.org/ - and linked from his articles, such Frances Coppola's on https://www.coppolacomment.com/

**Comment by rufuspollock** [comment]
@marypcbuk @spookylukey great suggestions and we now have this in as of https://github.com/rufuspollock/awesome-crypto-critique/commit/dda091225e7b6063f5b01fc681c1ecde4253c6cd plus follow up commits.

@spookylukey if you have specific selection of articles from blog.dshr.org that would be great - i've found one good one already.
